### PR TITLE
Add spatial examples (WIP)

### PR DESCRIPTION
**Pull request by dharhas**
These examples currenty work in the user-pangeo environment on https://jupyter.qhub.esipfed.org/

### CODE REVIEW

**Comment by dharhas** [comment]
@kcpevey these should work in the user-pangeo env

**Comment by kcpevey** [comment]
The git diff of `0x_Accessing_Landsat8_Collection2_Level2_Data.ipynb` is too big for gh so I can't comment directly on it. 

* I was able to run it with the `ogc` env [from here](https://github.com/Quansight/esip-ogc/pull/3). Didn't see any issues.
* this notebook needs to have output cleared
* The second cell had some commentary. I didn't know what itt meant so I just skipped over it and it didn't seem to be required. 
```
# Set the environment variable PC_SDK_SUBSCRIPTION_KEY, or set it here.
# The Hub sets PC_SDK_SUBSCRIPTION_KEY automatically.
# pc.settings.set_subscription_key(<YOUR API Key>)
```

**Comment by kcpevey** [comment]
the diff for `00x_Visualizing_Gridded_Data_WaveWatchIII.ipynb` is too big too. 

* It runs on `ogc`, no issues. 
* Needs to be cleared out

File: 00x_Exploring_The_National_Water_Model.ipynb [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "imax = ds[var].sel(time='2017-06-01 00:00:00').argmax().values"
```
Comment (kcpevey) [inline 00x_Exploring_The_National_Water_Model.ipynb:194]: this was the last line I was able to compute on my little laptop. Aside from not having enough memory locally, I didn't have any issues running this on the `ogc` env.

**Comment by dharhas** [comment]
closed in favor of #8

**Pull request closed without merging**